wireapp/wire-ios-mono
# refactor: show username screen on login

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

Show a screen for picking a username during the login flow if it's not already set. This previously handled using a takeover screen after the login was completed, that screen has now been removed.

Needs releases with:

- [x] https://github.com/wireapp/wire-ios/pull/863

### Testing

#### Test Coverage (Optional)


#### How to Test

Invite a new user to team and accept the invitation on the web but don't login in the webapp. Instead login via the iOS app and you will see a username screen after you've entered your credentials.

### Attachments

![D92A1561-565A-45BB-8CFC-F14BDC81B5DB](https://github.com/wireapp/wire-ios/assets/7156/29bad4fb-ffb8-465a-91b7-e673bcbce154)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
